CHANGES.md

```diff
@@ -15,6 +15,7 @@
 - Fixed debug label rendering in `Cesium3dTilesInspector`. [#10246](https://github.com/CesiumGS/cesium/issues/10246)
 - Fixed a crash that occurred in `ModelExperimental` when loading a Draco-compressed model with tangents. [#10294](https://github.com/CesiumGS/cesium/pull/10294)
 - Fixed an incorrect model matrix computation for `i3dm` tilesets that are loaded using `ModelExperimental`. [#10302](https://github.com/CesiumGS/cesium/pull/10302)
+- Fixed clamping issues with billboards after using mode "Disable depth test when clamped to ground." [#10191](https://github.com/CesiumGS/cesium/issues/10191)
```

Comment:
```suggestion
- Fixed race condition during billboard clamping when the height reference changes. [#10191](https://github.com/CesiumGS/cesium/issues/10191)
```

Reply:
Just a slight tweak to the message since the issue applies more broadly than just that sandcastle example.

Reply:
Sounds good! Thank you!